src/unix/linux_like/mod.rs

```diff
@@ -1776,8 +1728,65 @@ extern "C" {
     pub fn uname(buf: *mut ::utsname) -> ::c_int;
 }
 
+// LFS64 extensions
+//
+// * musl has 64-bit versions only so aliases the LFS64 symbols to the standard ones
+// * ulibc doesn't have preadv64/pwritev64 (TODO: Or does it?!)
```

Comment:
We prefer FIXME over TODO, and this is unclear what we should do, I think we could just drop that.

Reply:
Ah, whoops, that TODO was for me 😄 the existing code didn't declare `preadv64/pwritev64` on `ulibc` and I wanted to write a comment explaining why they were omitted here.  I've just had a look at indeed `ulibc` doesn't have these entrypoints (no idea why not) so the TODO can be removed.